### PR TITLE
feat(storefront): BCTHEME-600 Added translation files for Cornerstone

### PR DESCRIPTION
**Pull request by BC-tymurbiedukhin**
#### What?

We need to implement the language files provided in TRANS-844 into Cornerstone to meet CEV

#### Tickets / Documentation

- [Jira](https://jira.bigcommerce.com/browse/BCTHEME-600)

#### Notes
Every language has been tested in chrome at next pages: 
1.Home page
2.Category
3.Product
4.Account
5.Cart
6. AMP

### CODE REVIEW

File: lang/nl.json [hunk 1]
```diff
@@ -0,0 +1,955 @@
...
+    },
+    "admin": {
+        "maintenance_header": "Uw winkel is offline voor onderhoud.",
+        "maintenance_tooltip": "Alleen beheerders kunnen de winkel op dit moment bekijken. Ga naar de instellingenpagina van het configuratiescherm om de onderhoudsmodus uit te schakelen.",
+        "maintenance_showstore_link": "Klik hier om te zien wat uw bezoekers zien.",
+        "prelaunch_header": "Uw winkelontwerp is privé. Deel uw site met voorbeeldcode:",
+        "page_builder_link": "Deze pagina in Page Builder ontwerpen"
+    },
+    "carousel": {
+        "arrow_and_dot_aria_label": "Ga naar dia [NUMBER] van",
```
Comment (bc-alexsaiannyi) [inline lang/nl.json:904]: this construction has been affected in all languages. English equivalent is:  `"Go to slide [NUMBER] of" `
While original phrase from EN.json: `"Go to slide [SLIDE_NUMBER] of [SLIDES_QUANTITY]",`

File: lang/pt.json [hunk 2]
```diff
@@ -0,0 +1,955 @@
...
+        "slide_number": "Número do slide {slide_number}"
+    },
+    "validation_messages": {
+        "valid_email": "É necessário informar um email válido.",
+        "password": "É necessário informar uma senha.",
+        "password_match": "Suas senhas não correspondem.",
+        "invalid_password": "As senhas devem ter pelo menos sete caracteres e conter caracteres alfabéticos e numéricos.",
+        "field_not_blank": " o campo não pode ficar em branco.",
+        "certificate_amount": "É necessário informar um valor para o vale-presente.",
+        "certificate_amount_range": "É necessário informar um valor para o vale entre [MÍN] e [MÁX].",
```
Comment (bc-alexsaiannyi) [inline lang/pt.json:920]: MIN and MAX placeholders are also translated to local language.

**Comment by bc-alexsaiannyi** [comment]
**NOTE**: keys like `arrow_and_dot_aria_label`, `content_announce_message`, `certificate_amount_range` have been affected almost in all languages. For example, placeholder `[SLIDE_NUMBER]` has been separated and `slide` was removed from construction, while `[SLIDES_QUANTITY]` was removed from whole translation.  `[MIN]`, `[MAX]` placeholders were translated in `pt`, `pt-BR`.

File: lang/es.json [hunk 3]
```diff
@@ -0,0 +1,955 @@
...
+    },
+    "admin": {
+        "maintenance_header": "Su tienda está cerrada por mantenimiento.",
+        "maintenance_tooltip": "Solo los administradores pueden ver esta tienda en este momento. Vaya a la página de configuración para desactivar el modo de mantenimiento.",
+        "maintenance_showstore_link": "Haga clic aquí para ver lo que verán sus visitantes.",
+        "prelaunch_header": "Su tienda en línea es privada. Comparta su sitio con un código de vista previa:",
+        "page_builder_link": "Diseñar esta página en Page Builder"
+    },
+    "carousel": {
+        "arrow_and_dot_aria_label": "Ir a diapositiva [NUMBER] de",
```
Comment (bc-alexsaiannyi) [inline lang/es.json:904]: there is still a problem with placeholder. we can replace it with `Ir a diapositiva [SLIDE_NUMBER] de [SLIDES_QUANTITY]`

File: lang/es.json [hunk 4]
```diff
@@ -0,0 +1,955 @@
...
+        "active_dot_aria_label": "activo",
+        "content_announce_message": "Actualmente se encuentra en la diapositiva [NUMBER] de",
```
Comment (bc-alexsaiannyi) [inline lang/es.json:906]: the same here: `Actualmente se encuentra en la diapositiva [SLIDE_NUMBER] de [SLIDES_QUANTITY]`

File: lang/sv.json [hunk 5]
```diff
@@ -0,0 +1,1010 @@
...
+        "unsaved_changes": "Du har osparade ändringar",
+        "by_using": "Genom att använda vår webbplats godkänner du vår",
+        "agree_on_data_collection": "Genom att använda vår webbplats godkänner du att vi samlar in uppgifter om dig enligt beskrivningen i vår ",
+        "change_preferences": "Du kan ändra dina inställningar när som helst",
+        "cancel_dialog_title": "Är du säker på att du vill avbryta?",
+        "privacy_policy": "Sekretesspolicy",
+        "allow_category_tracking": "Tillåt [CATEGORY_NAME]-spårning",
+        "disallow_category_tracking": "Tillåt ej [CATEGORY_NAME]-spårning"
+    }
+}
```
Comment (yurytut1993) [inline lang/sv.json:1010]: can we add empty line here?

File: lang/es.json [hunk 6]
```diff
@@ -0,0 +1,955 @@
...
+        "field_not_blank": "El campo no puede dejarse en blanco.",
+        "certificate_amount": "Debe introducir un importe para el cupón de regalo.",
+        "certificate_amount_range": "Debe introducir un importe de certificado entre [MIN] y [MAX]",
+        "price_min_evaluation": "El precio mínimo debe ser menor que el precio máximo.",
+        "price_max_evaluation": "El precio mínimo debe ser menor que el precio máximo.",
+        "price_min_not_entered": "El campo de precio mínimo es obligatorio.",
+        "price_max_not_entered": "El campo de precio máximo es obligatorio.",
+        "price_invalid_value": "La entrada debe ser mayor que 0."
+    }
+}
```
Comment (bc-yevhenii-buliuk) [inline lang/es.json:955]: should we add a new line at the end of file?

File: lang/it.json [hunk 7]
```diff
@@ -0,0 +1,1010 @@
...
+        "unsaved_changes": "Sono state apportate modifiche non salvate",
+        "by_using": "Utilizzando il nostro sito, accetti i",
+        "agree_on_data_collection": "Utilizzando il nostro sito, acconsenti alla raccolta di dati come descritto nella nostra ",
+        "change_preferences": "Puoi modificare le tue preferenze in qualsiasi momento",
+        "cancel_dialog_title": "Vuoi davvero annullare?",
+        "privacy_policy": "Informativa sulla privacy",
+        "allow_category_tracking": "Consenti il tracciamento di [CATEGORY_NAME]",
+        "disallow_category_tracking": "Non consentire il tracciamento di [CATEGORY_NAME]"
+    }
+}
```
Comment (bc-yevhenii-buliuk) [inline lang/it.json:1010]: @BC-tymurbiedukhin should we add a new line at the end of file?

File: lang/nl.json [hunk 8]
```diff
@@ -0,0 +1,1010 @@
...
+        "unsaved_changes": "U heeft niet-opgeslagen wijzigingen",
+        "by_using": "Door onze website te gebruiken, gaat u akkoord met onze",
+        "agree_on_data_collection": "Door onze website te gebruiken, gaat u akkoord met het verzamelen van gegevens zoals beschreven in ons ",
+        "change_preferences": "U kunt uw voorkeuren op elk moment wijzigen",
+        "cancel_dialog_title": "Weet u zeker dat u uw instellingen wilt annuleren?",
+        "privacy_policy": "Privacybeleid",
+        "allow_category_tracking": "Tracking van [CATEGORY_NAME] toestaan",
+        "disallow_category_tracking": "Tracking van [CATEGORY_NAME] weigeren"
+    }
+}
```
Comment (bc-yevhenii-buliuk) [inline lang/nl.json:1010]: @BC-tymurbiedukhin should we add a new line at the end of file?

File: lang/pt-BR.json [hunk 9]
```diff
@@ -0,0 +1,1010 @@
...
+        "unsaved_changes": "Algumas alterações não foram salvas",
+        "by_using": "Ao utilizar o nosso site, você concorda com nosso(a)",
+        "agree_on_data_collection": "Ao utilizar o nosso site, você concorda com a coleta de dados conforme descrito em nosso(a) ",
+        "change_preferences": "Você pode alterar as suas preferências quando quiser",
+        "cancel_dialog_title": "Deseja mesmo cancelar?",
+        "privacy_policy": "Política de privacidade",
+        "allow_category_tracking": "Permitir monitoramento de [CATEGORY_NAME]",
+        "disallow_category_tracking": "Não permitir monitoramento de [CATEGORY_NAME]"
+    }
+}
```
Comment (bc-yevhenii-buliuk) [inline lang/pt-BR.json:1010]: @BC-tymurbiedukhin should we add a new line at the end of file?

File: lang/pt.json [hunk 10]
```diff
@@ -0,0 +1,1010 @@
...
+        "unsaved_changes": "Algumas alterações não foram salvas",
+        "by_using": "Ao utilizar o nosso site, você concorda com nosso(a)",
+        "agree_on_data_collection": "Ao utilizar o nosso site, você concorda com a coleta de dados conforme descrito em nosso(a) ",
+        "change_preferences": "Você pode alterar as suas preferências quando quiser",
+        "cancel_dialog_title": "Deseja mesmo cancelar?",
+        "privacy_policy": "Política de privacidade",
+        "allow_category_tracking": "Permitir monitoramento de [CATEGORY_NAME]",
+        "disallow_category_tracking": "Não permitir monitoramento de [CATEGORY_NAME]"
+    }
+}
```
Comment (bc-yevhenii-buliuk) [inline lang/pt.json:1010]: @BC-tymurbiedukhin should we add a new line at the end of file?

File: lang/es.json (hunk 3 above)
Comment (BC-tymurbiedukhin) [reply lang/es.json]: updated.

File: lang/es.json (hunk 4 above)
Comment (BC-tymurbiedukhin) [reply lang/es.json]: updated.

File: lang/sv.json (hunk 5 above)
Comment (BC-tymurbiedukhin) [reply lang/sv.json]: updated.

File: lang/es.json (hunk 6 above)
Comment (BC-tymurbiedukhin) [reply lang/es.json]: updated.

File: lang/it.json (hunk 7 above)
Comment (BC-tymurbiedukhin) [reply lang/it.json]: updated.

File: lang/nl.json (hunk 8 above)
Comment (BC-tymurbiedukhin) [reply lang/nl.json]: updated.

File: lang/pt-BR.json (hunk 9 above)
Comment (BC-tymurbiedukhin) [reply lang/pt-BR.json]: updated.

File: lang/pt.json (hunk 10 above)
Comment (BC-tymurbiedukhin) [reply lang/pt.json]: updated.

**Review by bc-alexsaiannyi — APPROVED** [review]

**Review by bc-yevhenii-buliuk — APPROVED** [review]

**Review by yurytut1993 — APPROVED** [review]

**Pull request merged**